_posts/en/newsletters/2024-10-18-newsletter.md

```diff
@@ -0,0 +1,64 @@
+---
+title: 'Bitcoin Optech Newsletter #325'
+permalink: /en/newsletters/2024/10/18/
+name: 2024-10-18-newsletter
+slug: 2024-10-18-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME:harding
+
+## News
+
+_No significant news this week was found in any of our [sources][optech sources]._
```

Comment:
https://delvingbitcoin.org/t/ln-summit-2024-notes-summary-commentary/1198 just dropped, FYI, if there is time

Reply:
Added!